### PR TITLE
Result of lesson 11.

### PR DESCRIPTION
**Pull request by ssaleta**

### CODE REVIEW

**Comment by ssaleta** [comment]
@gradopaw lesson 11 👍

File: students/saleta_sebastian/lesson_11_argparse_and_math/regex_search.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import argparse
+import os
+import sys
+import re
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Script to find sentence'
+                                                 ' in txt files ')
+    parser.add_argument('-p', '--sentence',
```
Comment (gradopaw) [inline students/saleta_sebastian/lesson_11_argparse_and_math/regex_search.py]: Shouldn't be ```parser.add_argument('-s', '--sentence',```

As I know you from this course you would like to fix this, am I wrong?

Comment (ssaleta) [reply students/saleta_sebastian/lesson_11_argparse_and_math/regex_search.py]: I love mistakes like this 👍

**Comment by ssaleta** [comment]
@gradopaw ASAP fix

**Pull request merged**